tests/test-cases/Group1-Docker-Commands/1-33-Docker-Service.robot

```diff
@@ -42,12 +42,12 @@ Docker serivce rm
 Docker service scale
     ${rc}  ${output}=  Run And Return Rc And Output  docker1.13 %{VCH-PARAMS} service scale test-service=3
```

Comment:
Yeah: `docker service scale <id of service>=<number of tasks>`. 

https://docs.docker.com/engine/swarm/swarm-tutorial/scale-service/  for more info